ui/lib/config-ui/addon/routes/login-settings/rule/details.js

```diff
@@ -4,5 +4,26 @@
  */
 
 import Route from '@ember/routing/route';
+import { service } from '@ember/service';
 
-export default class LoginSettingsRuleDetailsRoute extends Route {}
+export default class LoginSettingsRuleDetailsRoute extends Route {
+  @service api;
```

Comment:
This is the new service jordan is giving us. But I don't see it used here, we're still using the store (ember data). I don't think you need this?